build/config.js

```diff
@@ -137,6 +137,9 @@ let BuildConfig = function(config) {
     compilation_level: 'SIMPLE_OPTIMIZATIONS',
   };
 
+  /** @type {!object} */
+  this.closureStylesheetsCompilerOptions = this.options.gss || {};
```

Comment:
here is the change in `build/config.js`, to line up with [closure-builder.js](https://github.com/google/closure-builder/blob/master/closure-builder.js#L315), which is enclosed below for reference:

```js
/**
 * @param {!buildConfig} config
 * @param {function=} opt_callback
 */
ClosureBuilder.prototype.compileClosureStylesheetsFiles = function(
    config, opt_callback) {
  config.setMessage('Compiling Closure Stylesheets files ...');
  let files = config.getClosureStylesheetsFiles();
  buildCompilers.compileClosureStylesheetsFiles(
    files, config.getOutFilePath(), {
      config: config,
      options: config.closureStylesheetsCompilerOptions,
    },
    opt_callback);
};
```